support-docs/README.md

```diff
@@ -0,0 +1,107 @@
+May: Indicates an optional requirement.
+
+1.4	References
+1)	The JSON data interchange syntax, ECMA-404, 2nd edition, December 2017 
+2)	JSON Schema: A media Type for Describing JSON Documents, draft-bhutton-json-schema-00, December 2020.
+3)	Terms, Definitions, and Letter Symbols for Microelectronic Devices, JESD99C – December 2012
+
+2	Use Cases 
+Digital datasheets can be used for many applications. The list of use cases included here is not meant to be exhaustive and it is expected that new applications will be developed as more people start using these digital datasheets. Sample applications include automated hardware design checks to identify bugs earlier in the design cycle, automated hardware designs to speed up board development, components comparison to identify replacement components on a design. 
+
+3	Specifications
+3.1	Digital Datasheet Format
+A digital datasheet shall be written in the JSON (JavaScript Object Notation) format. The specification is written in JSON schema to facilitate validation of a JSON datasheet. 
+3.2	Required Information.
+A digital datasheet shall include the following information:
+The manufacturer’s name
+The Manufacturer Part Number (MPN)
+The datasheet release date
+The datasheet version
+The component part type. See appendix for examples.
+List of Component Pins as defined by the specification.
+
+3.3	Date Format
+A digital datasheet shall follow the international standard notation, YYYY-MM-DD.
+
+3.4	Units
+The digital datasheet shall follow the International System of Units, SI except for specific cases, like package dimensions, where the imperial system of units may be used. These specific cases will be addressed in the specifications.
+Properties used the represent units in the datasheets are listed in Github: 
+They include:
+|Unit Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|siUnit|Name of the SI unit of measure|String|Yes|
+|absoluteValue|Unit quantity corresponding to unit text|Number|Yes|
+|unitText|Human readable text describing value|String|Yes|
+|unitFactor|Multiplier on the value to achieve the SI unit|Number|Yes|
+|relativeValueReference|if unit quantity is based on another reference, value of the reference|String| |
+|relativeValueModifier|if a unit quantity is based on another reference, the value that edits that reference|Number| |
+|relativeValueOperator|if a unit quantity is based on another reference, the operation that is performed with the modifier|String| |
+|valueDefined|a boolean representing whether a value has been defined|Boolean| |
+
+3.5	Terms and Definitions
+The terms and definitions used for electronic components shall follow the JEDEC standards.
+
+3.6	Missing Values
+Missing fields shall be represented by the NULL character.
+
+3.7	Pins Specifications
+The pins specification is included in Github at:
+Table below shows the list of properties used to describe pins
+
+|Pin Property|Description|Data Type|Required or Optional|

```

Comment:
Change

"Required or Optional" to "Required?"

Reply:
Changed